### ISSUE TITLE
Fix git-commit-id-plugin version

### ISSUE THREAD

**Issue by philwebb** [spring-projects/spring-boot#7926]
It's declared in `spring-boot-starter-parent` and `spring-boot-dependencies` with different versions.

```xml
				<plugin>
					<groupId>pl.project13.maven</groupId>
					<artifactId>git-commit-id-plugin</artifactId>
					<version>2.1.11</version>
```

**Comment by tjuchniewicz** [comment]
We need update this plugin to the latest version. See #7980.

**Comment by wilkinsona** [comment]
The version in `spring-boot-dependencies` (2.1.11) wins. We should stick with 2.1.x in 1.4 and upgrade to 2.2.x in 1.5. People using Boot 1.4 and affected by ktoso/maven-git-commit-id-plugin#240 can override the version.

**Comment by wilkinsona** [comment]
Closed by f92719c4beffd8dc313604c8ccdb9f8857359967
